chat-plugins/scavengers.js

```diff
@@ -1157,43 +1177,65 @@ let commands = {
 	setblitz: function (target, room, user) {
 		if (room.id !== 'scavengers') return this.errorReply("This command can only be used in the scavengers room.");
 		if (!this.can('mute', null, room)) return false; // perms for viewing only
-		if (!target) return this.sendReply(`The points rewarded for winning official hunts is: ${(room.blitzPoints || DEFAULT_BLITZ_POINTS)}`);
+
+		if (!target) {
+			let points = [];
+			const source = Object.entries(Object.assign(DEFAULT_BLITZ_POINTS, room.blitzPoints || {}));
+			for (const entry of source) {
+				points.push(`${entry[0]}: ${entry[1]}`);
+			}
+			return this.sendReplyBox(`The points rewarded for winning hunts is:<br />${points.join('<br />')}`);
+		}
 
 		if (!this.can('declare', null, room)) return false; // perms for editing
 
-		let blitzPoints = parseInt(target);
-		if (isNaN(blitzPoints) || blitzPoints < 0 || blitzPoints > 1000) return this.errorReply("The points value awarded for blitz must be an integer bewteen 0 and 1000.");
+		let [type, blitzPoints] = target.split(',');
+		blitzPoints = parseInt(blitzPoints);
+		type = toId(type);
 
-		room.blitzPoints = blitzPoints;
+		if (isNaN(blitzPoints) || blitzPoints < 0 || blitzPoints > 1000) return this.errorReply("The points value awarded for blitz must be an integer bewteen 0 and 1000.");
+		if (!room.blitzPoints) room.blitzPoints = {};
+		room.blitzPoints[type] = blitzPoints;
 
 		if (room.chatRoomData) {
 			room.chatRoomData.blitzPoints = room.blitzPoints;
 			Rooms.global.writeChatRoomData();
 		}
-		this.privateModAction(`(${user.name} has set the points awarded for blitz to ${blitzPoints}.)`);
-		this.modlog('SCAV BLITZ', null, blitzPoints);
+		this.privateModAction(`(${user.name} has set the points awarded for blitz for ${type} hunts to ${blitzPoints}.)`);
+		this.modlog('SCAV BLITZ', null, `${type}: ${blitzPoints}`);
 	},
 
 	setpoints: function (target, room, user) {
 		if (room.id !== 'scavengers') return this.errorReply("This command can only be used in the scavengers room.");
 		if (!this.can('mute', null, room)) return false; // perms for viewing only
-		if (!target) return this.sendReply(`The points rewarded for winning official hunts is: ${(room.winPoints || DEFAULT_POINTS).map((p, i) => `(${(i + 1)}) ${p}`).join(', ')}`);
+
+		if (!target) {
+			let points = [];
+			const source = Object.entries(Object.assign(DEFAULT_POINTS, room.winPoints || {}));
+			for (const entry of source) {
+				points.push(`${entry[0]}: ${entry[1].map((p, i) => `(${(i + 1)}) ${p}`).join(', ')}`);
+			}
+			return this.sendReplyBox(`The points rewarded for winning hunts is:<br />${points.join('<br />')}`);
+		}
 
 		if (!this.can('declare', null, room)) return false; // perms for editting
 
-		let winPoints = target.split(',').map(p => parseInt(p));
+		let [type, ...pointsSet] = target.split(',');
+		type = toId(type);
```

Comment:
Should we check that `type` is something we actually want to set (e.g. official or regular)? So we don't accidentally start setting the points for practice/unrated.